msu/utils/utils.nut

```diff
@@ -244,6 +245,12 @@
 		}
 	}
 
+	// Private
+	function __generateUID()
+	{
+		return ::MSU.Utils.UID++;
+	}
+
```

Comment:
Should this be private? Should other mods not also be able to use this?

Reply:
This is just for safety. I want to make sure that it isn't used in any way that can lead to the whole system being messed up.

Reply:
ok, keep it like that for now I suppose.